### ISSUE TITLE
GatedRecurrent has no biases

### ISSUE THREAD

**Issue by bartvm** [bartvm/blocks#440]
@rizar Correct me if I'm wrong, but I just noticed that the `GatedRecurrent` bricks doesn't have biases implemented. Is that by design or just an omission?

**Comment by rizar** [comment]
This is sort by of by design, because I assumed that the input has biases, in which case inner biases would be redundant. I guess it should mentioned in the documentation.

**Comment by bartvm** [comment]
Yeah, also, can't the same trick be applied to `LSTM` in that case? That's what confused me actually, I noticed that `LSTM` did have them but `GatedRecurrent` didn't.
